pkg/services/accesscontrol/roles.go

```diff
@@ -15,19 +15,23 @@ type RoleRegistry interface {
 
 // Roles definition
 var (
-	datasourcesEditorReadRole = RoleDTO{
-		Version: 1,
-		Name:    datasourcesEditorRead,
+	datasourcesQuerierRole = RoleDTO{
+		Version:     2,
+		Name:        datasourcesQuerier,
+		DisplayName: "Data sources querier",
+		Description: "Query data sources using the Explore feature in Grafana. Data source permissions still apply to users with this role, so they will only be able to query datasources for which they have query permissions.",
```

Comment:
I found this slightly difficult to understand, maybe it could be shortened a bit to something like 
```suggestion
		Description: "Query data sources using the Explore feature in Grafana. Users will only be able to query datasources for which they also have data source query permissions.",
```